scripts/ci_setup.py

```diff
@@ -355,6 +361,8 @@ def __main(args: list) -> int:
             out_file.write(variable_format % ('UseSharedCompilation', 'false'))
             out_file.write(variable_format % ('DOTNET_ROOT', dotnet_path))
             out_file.write(variable_format % ('MAUI_VERSION', args.maui_version))
+            if args.affinity:
```

Comment:
@LoopedBard3 or @cincuranet - can one of you confirm how does this get used to set the affinity during benchmark launch?

Reply:
[This line](https://github.com/dotnet/performance/pull/3020/files#diff-744fe116e315e84456d3107734f7b9f7f57bd567a6599a215a423cb63cf01cf7R97) sets the [parameter](https://github.com/dotnet/performance/pull/3020/files#diff-744fe116e315e84456d3107734f7b9f7f57bd567a6599a215a423cb63cf01cf7R28) for BDN [via command line](https://benchmarkdotnet.org/api/BenchmarkDotNet.ConsoleArguments.CommandLineOptions.html#BenchmarkDotNet_ConsoleArguments_CommandLineOptions_Affinity).

Reply:
@adamsitnik - I checked the BDN code, but just want to confirm my understanding. We launch the process, set the affinity mask and then start the benchmark execution, right? Because, `ProcessorAffinity` field of `Process` object can only be set after the process is launched.

Reply:
> We launch the process, set the affinity mask and then start the benchmark execution, right?

@kunalspathak that is correct:

start->priority->affinity

https://github.com/dotnet/BenchmarkDotNet/blob/40305b402ff795992ac9d926c8ad1aee853d8492/src/BenchmarkDotNet/Toolchains/Executor.cs#L87